airbyte-webapp/src/components/connection/DestinationNamespaceModal/ExampleSettingsTable.module.scss

```diff
@@ -1,50 +1,52 @@
-@use "scss/colors";
-@use "scss/variables";
-
-.exampleSettingsTable {
-  border-spacing: 0;
-  border-collapse: separate;
-
-  tr {
-    th, td {
-      padding: variables.$spacing-md;
-      border-style: solid;
-      border-color: colors.$dark-blue-50;
-      border-width: 0 variables.$border-thin variables.$border-thin 0;
-    }
-
-    th {
-      border-top-width: variables.$border-thin;
-      background-color: colors.$dark-blue-50;
-      text-align: left;
-
-      .text {
-        color: colors.$grey-600;
-      }
-    }
-
-    td {
-      background-color: colors.$white;
-
-      &:first-child, th:first-child {
-        border-left-width: variables.$border-thin;
-      }
-    }
-
-    &:first-child th:first-child {
-      border-top-left-radius: variables.$border-radius-xs;
-    }
-
-    &:first-child th:last-child {
-      border-top-right-radius: variables.$border-radius-xs;
-    }
-
-    &:last-child td:first-child {
-      border-bottom-left-radius: variables.$border-radius-xs;
-    }
-
-    &:last-child td:last-child {
-      border-bottom-right-radius: variables.$border-radius-xs;
-    }
-  }
-}
+@use "scss/colors";
+@use "scss/variables";
+
+.exampleSettingsTable {
+  border-spacing: 0;
+  border-collapse: separate;
+
+  tr {
+    th,
+    td {
```

Comment:
So I merged this in from master, and apparently my prettier settings conflict with whatever was there (`th, td` got moved to two lines). But I'm not sure why this whole file is showing up in the diff... Is this a GitHub bug or am I missing something here...?

Reply:
Update: it's because these files were commited with CRLF terminators (probably from a windows machine?) and I assume prettier converted them to just LF on my unix machine (macbook).

<img width="602" alt="Screenshot 2022-12-09 at 9 27 32 PM" src="https://user-images.githubusercontent.com/7550957/206790722-d614270a-d983-48e9-ae08-c07ffb41d0dc.png">

Reply:
I opened a separate PR to fix these changes here: https://github.com/airbytehq/airbyte/pull/20370

Hopefully we can get that merged first, and this will disappear from the diff in this PR 😄